website/server/controllers/api-v3/auth.js

```diff
@@ -130,6 +130,14 @@ api.registerLocal = {
       newUser.registeredThrough = req.headers['x-client']; // Not saved, used to create the correct tasks based on the device used
     }
 
+    // A/B Test 2016-09-12: Start with Sound Enabled?
+    if (Math.random() < 0.5) {
+      newUser.preferences.sound = 'rosstavoTheme';
+      newUser._ABtest = '20160912-soundEnabled';
+    } else {
+      newUser._ABtest = '20160912-soundDisabled';
+    }
+
```

Comment:
Agreed. Since this is all private, we don't have to worry about breaking changes. (Before, when we were going to expose it to the API, we would have needed to worry about it)